### PR TITLE
botsfordiscord.com added

### PR DESCRIPTION
**Pull request by brisdevs**

### CODE REVIEW

File: websites/B/Bots For Discord/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/bk2PPfW.png",
+  "thumbnail": "https://i.imgur.com/skUsdmA.png",
+  "color": "#7289da",
+  "tags": [
+    "discord",
+    "discord-bot",
+    "discord-bot-list"
+  ],
+
```
Comment (Bas950) [inline websites/B/Bots For Discord/dist/metadata.json:23]: remove the extra enter

File: websites/B/Bots For Discord/presence.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+    clientId: "806926545771167774"
+});
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo"
+    },
+        browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/B/Bots For Discord/presence.ts:9]: you should probs init it outside of the updatedata event.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/Bots For Discord/dist/metadata.json (hunk 1 above)
Comment (brisdevs) [reply websites/B/Bots For Discord/dist/metadata.json]: Fixed 👌

File: websites/B/Bots For Discord/presence.ts (hunk 2 above)
Comment (brisdevs) [reply websites/B/Bots For Discord/presence.ts]: Fixed 👌

File: websites/B/Bots For Discord/presence.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        } else if (window.location.pathname.startsWith('/tag/')) {
+            presenceData.details = "Viewing bots with tag:";
+            presenceData.state = document.querySelector('div.container h1').innerHTML.split('</i>')[1].trim();
+        } else if (window.location.pathname.toLowerCase() == '/login') {
+            presenceData.state = 'Login';
+        } else if (window.location.pathname.startsWith("/bot/")) {
+            presenceData.details = 'Viewing a bot:';
+            presenceData.state = document.querySelectorAll("div.container #highlight")[0].innerHTML.split("<a")[0].trim();
+        }
+    } if (window.location.hostname == 'docs.botsfordiscord.com') {
```
Comment (ririxidev) [inline websites/B/Bots For Discord/presence.ts:52]: borked. should be ``} else if {``

File: websites/B/Bots For Discord/presence.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        largeImageKey: "logo"
+    },
+    browsingStamp = Math.floor(Date.now() / 1000);
+    presenceData.startTimestamp = browsingStamp;
+    if (window.location.hostname == 'botsfordiscord.com') {
+        presenceData.details = "Viewing page:";
+        if (document.querySelector('head > title').innerHTML.split("|")[0].trim().startsWith('Error')) {
+            presenceData.state = document.querySelector('#errortitle').innerHTML.trim() + ' (' + document.querySelector('#errorsubtitle').innerHTML.trim() + ')';
+        }
+        if (window.location.pathname.toLowerCase() == "/") {
```
Comment (ririxidev) [inline websites/B/Bots For Discord/presence.ts:16]: do else if or something pls

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/B/Bots For Discord/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "bris",
+    "id": "656165298818514946"
+  },
+  "service": "BotsForDiscord",
```
Comment (Bas950) [inline websites/B/Bots For Discord/dist/metadata.json:6]: Service name and folder name must be the same, spaces caps etc. included

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

File: websites/B/Bots For Discord/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "bris",
+    "id": "656165298818514946"
+  },
+  "service": "Bots For Discord",
+  "description": {
+    "en": "Bots for Discord a page listing the bots for Discord!"
```
Comment (Bas950) [inline websites/B/Bots For Discord/dist/metadata.json:8]: Use the official description;
`Find the perfect discord bot for your server on Bots For Discord, or list your bot for others to find.`

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/Bots For Discord/dist/metadata.json (hunk 6 above)
Comment (brisdevs) [reply websites/B/Bots For Discord/dist/metadata.json]: Edited 👌

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**